### PR TITLE
add search_facts tests for organization and params

### PR DESCRIPTION
**Pull request by evgeni**
Fixes: #574

### CODE REVIEW

File: tests/test_playbooks/search_facts.yml [hunk 1]
```diff
@@ -3,10 +3,21 @@
   gather_facts: false
   vars_files:
     - vars/server.yml
+    - vars/search_facts.yml
   tasks:
     - include: tasks/organization.yml
       vars:
+        organization_name: "{{ item.organization }}"
         organization_state: present
+      loop: "{{ test_resources }}"
```
Comment (mdellweg) [inline tests/test_playbooks/search_facts.yml]: I am not sure, i understand the purpose of the `test_reseources` variable, but should we add a `when: item.organization` here?

Comment (evgeni) [reply tests/test_playbooks/search_facts.yml]: oh the purpose was purely that I don't have to write individual create/destroy calls and not to repeat the same data multiple times.

IMHO item.org will always be set, so no need for the when

Comment (mdellweg) [reply tests/test_playbooks/search_facts.yml]: As the list is rather short, we could just explode it here...

Comment (evgeni) [reply tests/test_playbooks/search_facts.yml]: I'd prefer to keep it in a vars file. Would a different variable name help in any way?

**Pull request merged**